plotly/offline/offline.py

```diff
@@ -140,7 +140,11 @@ def init_notebook_mode(connected=False):
             '</script>'
             '').format(script=get_plotlyjs())
 
-    ipython_display.display(ipython_display.HTML(script_inject))
+    display_bundle = {
+        'text/html': script_inject,
+        'text/vnd.plotly.v1+html': script_inject
+    }
+    ipython_display.display(display_bundle, raw=True)
```

Comment:
@Kully : 

From IPython Docs:

> raw : bool
Are the data objects raw data or Python objects that need to be formatted before display? [default: False]

We dont want IPython to do any further processing on this, so we use `raw=True`.